### PR TITLE
:arrow_up: Update to node-emoji for unicode 13.1 emoji support

### PR DESCRIPTION
**Pull request by DanielHilton**
And also remove npm audit vulns + update ava

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,21 +48,17 @@
...
-    "node-emoji": "^1.10.0"
+    "node-emoji": "^1.11.0"
   },
   "peerDependencies": {
     "semantic-release": "^17.0.0"
-  },
-  "ava": {
-    "babel": false,
-    "compileEnhancements": false
   }
```
Comment (DanielHilton) [inline package.json:63]: not required by ava 3.x.x

**Review by momocow — APPROVED** [review]
Thanks for your PR!

**Comment by momocow** [comment]
Oops, conflicts appear in package-lock.json.

Would you like to redo and commit the package-lock.json?

Thank you :)

**Comment by DanielHilton** [comment]
@momocow sorted it 👍🏻

**Comment by momocow** [comment]
Released in v1.4.0.

**Pull request merged**